content/en/integrations/guide/prometheus-metrics.md

```diff
@@ -7,65 +7,139 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/sql-server-metrics/#create-a-stored-procedure-to-generate-and-collect-metrics"
   tag: "Blog"
   text: "Create a stored procedure to generate and collect metrics"
+- link: "/integrations/openmetrics/"
+  tag: "Documentation"
+  text: "Learn about the OpenMetrics integration"
 - link: "/integrations/mysql/"
   tag: "Documentation"
-  text: "Datadog-MySQL integration"
+  text: "Learn about the MySQL integration"
 - link: "/agent/kubernetes/prometheus/"
   tag: "Documentation"
   text: "Kubernetes Prometheus and OpenMetrics metrics collection"
 ---
 
-If you are using Datadog's Prometheus or OpenMetrics checks, you may be interested in how these metrics map to existing Datadog metrics types.
+## Overview
 
-See [Kubernetes Prometheus and OpenMetrics Metrics Collection][1] for more information.
+This page walks you through how Prometheus or OpenMetrics check metrics map to existing Datadog metric types.
 
 ## Prometheus and OpenMetrics metric types
 
-* `counter`: a cumulative metric that represents a single monotonically increasing counter, whose value can only increase—or be reset to zero.
-* `gauge`: a metric that represents a single numeric value, which can arbitrarily go up and down.
-* `histogram`: samples observations and counts them in configurable buckets; also provides a sum of all observed values.
-* `summary`: similar to histogram; samples observations, provides a sum of all observed values, and calculates configurable quantiles over a sliding time window.
+For more information, see [Kubernetes Prometheus and OpenMetrics Metrics Collection][1].
+
+{{< tabs >}}
+{{% tab "Latest Version" %}}
+* `gauge`: A metric that represents a single numeric value, which can arbitrarily go up and down.
+* `counter`: A cumulative metric that represents a single monotonically increasing counter, whose value can only increase—or be reset to zero.
+* `stateset`: A metric that represents a series of related boolean values, also known as a bitset. 
+* `info`: A metric that exposes textual information which should not change during the process lifetime. For example, an application's version, revision control commit, and the compiler version.
+* `histogram`: Samples observations and counts them in configurable buckets; also provides a sum of all observed values.
+* `gaugehistogram`: Samples current distributions and counts them in configurable buckets; also provides a sum of all observed values.
```

Comment:
We don't support `stateset`, `info` and `gaugehistogram` currently (and will probably do so when there's demand for it), so I think we should leave those out. Note that https://prometheus.io/docs/concepts/metric_types/ doesn't mention them either.

Reply:
```suggestion
* `histogram`: Samples observations and counts them in configurable buckets; also provides a sum of all observed values.
```